README.md

```diff
@@ -17,39 +147,49 @@ git clone https://github.com/sparkletown/sparkle
 cd sparkle
 ```
 
-In the main `sparkle` folder, copy `.env.example` to a new file `.env.local`, and fill it with the appropriate details.
-
-You can read more about the various `.env` files that you can use at:
-
-- https://create-react-app.dev/docs/adding-custom-environment-variables#adding-development-environment-variables-in-env
+### Step 2: Install Packages
 
 Install the platform dependencies with `npm`:
 
-(**Note:** `npm` v7+ is not supported, it will cause issues with our `package-lock.json`, and you may end up with the wrong dependency versions)
+**Important Note:** `npm` v7+ is not supported, it will cause issues with our `package-lock.json`, and you may end up with the wrong dependency versions. Check your current npm version with `npm -v`.
```

Comment:
"Important" or "Note". Choose one. Both is overkill.

Reply:
fix 6c6c049